### PR TITLE
Proper description for Linux packages

### LINKED ISSUE #3209: truncated description in .deb installer
Screenshot_20230422_103846

### PR DESCRIPTION
**Pull request by farooqkz**
Fixes #3209

### CODE REVIEW

**Review by Simon-Laux — APPROVED** [review]

**Comment by Simon-Laux** [comment]
maybe also wait for ads review? but might be ok to merge if you tested it already with the GUI program that had the problem. 🤷

**Comment by adbenitez** [comment]
how do I download the .deb generated by this PR? IIRC they used to be ready for download clicking "details" but that doesn't seem to work

File: electron-builder.json5
Comment (Simon-Laux) [inline electron-builder.json5:414]: nope this is not right place for it, look at the first line, this file is auto generated

**Review by Simon-Laux — CHANGES_REQUESTED** [review]
cool that you found this fix, but it will not persist as you added it to an autogenerated file. you need to add it to `build/gen-electron-builder-config.js` instead and then run `npm run pack:generate_config` to update this autogenerated file

**Comment by farooqkz** [comment]
@adbenitez @Simon-Laux Yep I am strong with the Source :D

**Review by Simon-Laux — APPROVED** [review]

**Pull request merged**
